docs/rest_api/index.rst

```diff
@@ -0,0 +1,20 @@
+Bodhi REST API
+==============
+
+This section of the documentation describes Bodhi's REST API. You can read about the various
+sections of the API by following the links below:
+
+
+.. toctree::
+   :maxdepth: 2
+
+   builds
```

Comment:
You could just put all the REST API in this file (and maybe rename it to rest_api.rst). You'll still have a TOC to navigate to a section, but you don't have to click back and forth so much. I like it more in one file, but it's up to you.